src/backend/distributed/commands/multi_copy.c

```diff
@@ -631,7 +651,10 @@ CopyToNewShards(CopyStmt *copyStatement, char *completionTag, Oid relationId)
 										copyStatement->options);
 
 	CopyOutState copyOutState = (CopyOutState) palloc0(sizeof(CopyOutStateData));
-	copyOutState->binary = true;
+	copyOutState->delim = delimiterCharacter;
+	copyOutState->null_print = nullPrintCharacter;
+	copyOutState->null_print_client = nullPrintCharacter;
```

Comment:
On 2016-07-12 08:21:18 -0700, Brian Cloutier wrote:
> > @@ -631,7 +651,10 @@ CopyToNewShards(CopyStmt *copyStatement, char *completionTag, Oid relationId)
> >  										copyStatement->options);
> >  
> >  	CopyOutState copyOutState = (CopyOutState) palloc0(sizeof(CopyOutStateData));
> > -	copyOutState->binary = true;
> > +	copyOutState->delim = delimiterCharacter;
> > +	copyOutState->null_print = nullPrintCharacter;
> > +	copyOutState->null_print_client = nullPrintCharacter;
> 
> More compiler warnings:
> 
> ```
> commands/multi_copy.c: In function ‘CopyToNewShards’:
> commands/multi_copy.c:654:22: warning: assignment discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
>   copyOutState->delim = delimiterCharacter;
>                       ^
> commands/multi_copy.c:655:27: warning: assignment discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
>   copyOutState->null_print = nullPrintCharacter;
>                            ^
> commands/multi_copy.c:656:34: warning: assignment discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
>   copyOutState->null_print_client = nullPrintCharacter;
>                                   ^
> ```

Burak, what compiler do you use? These are pretty common warnings, and
IIRC are automatically enabled by our configure script?